assets/src/application/ui/display/TimezoneTimeInfo/style.scss

```diff
@@ -1,11 +1,30 @@
-.ee-timezone-info.ee-timezone-info {
+.ee-timezone-info {
 	display: inline-block;
 	margin: 0 auto;
 	position: relative;
 
+	&__button.ee-btn-base {
+		border: none;
+		font-size: 16px;
+		height: 30px;
+		margin: var(--ee-margin-micro);
+		min-width: 30px;
+		padding: 4px;
+		width: 30px;
+	}
+
+	&__icon {
+		height: 16px;
+		margin: 0;
```

Comment:
doesn't seem to have any effects, is this fixing smth?

Reply:
there _was_ a margin being applied to that icon that this was removing... ???

Reply:
you are right, that has been applied via `ee-svg`, my bad